### PR TITLE
answered the prompt

### PR DESCRIPTION
**Pull request by Lauren34**

### CODE REVIEW

File: src/main/java/Lesser/finalproject/Json/ArtObject.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package Lesser.finalproject.Json;
```
[the remark this answers is not in the document]
Comment (andrewoid) [reply src/main/java/Lesser/finalproject/Json/ArtObject.java]: Package names must be in lower case.

Copy all the Java files out of your repo. Delete everything. Add the deletions and then put the files back in the correct package names and directories.

File: src/test/java/Lesser/finalproject/RijksServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Lesser.finalproject.Json.CollectionResponse;
+import com.andrewoid.ApiKey;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RijksServiceTest {
+
+    @Test
+    public void testPageNumber() {
```
Comment (andrewoid) [inline src/test/java/Lesser/finalproject/RijksServiceTest.java]: Do not have the word test in front of all your methods.

File: src/test/java/Lesser/finalproject/RijksServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    @Test
+    public void testSearchCollection() {
+        ApiKey apiKey = new ApiKey();
+        RijksService service = new RijksServiceFactory().getService();
+
+        CollectionResponse response = service.searchCollection("Rembrandt", 1, apiKey.get()).blockingGet();
+
+        assertNotNull(response);
+        assertNotNull(response.artObjects[0]);
```
Comment (andrewoid) [inline src/test/java/Lesser/finalproject/RijksServiceTest.java]: Assert that there's data in the art objects?

File: src/test/java/lesser/finalproject/RijksServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijksService service = new RijksServiceFactory().getService();
+
+        // when
+        CollectionResponse response = service.pageNumber(1, apiKey.get()).blockingGet();
+
+        // then
+        assertNotNull(response);
+        assertNotNull(response.artObjects[0]);
```
Comment (andrewoid) [inline src/test/java/lesser/finalproject/RijksServiceTest.java]: Assert the values inside of artobject

File: src/test/java/lesser/finalproject/RijksServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Test
+    public void searchByArtist() {
+        // given
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijksService service = new RijksServiceFactory().getService();
+
+        // when
+        CollectionResponse response = service.searchByArtist("Vermeer", 1, apiKey.get()).blockingGet();
```
Comment (andrewoid) [inline src/test/java/lesser/finalproject/RijksServiceTest.java]: You need to use the artist's full name

**Review by andrewoid — APPROVED** [review]

**Comment by andrewoid** [comment]
Why haven't you merged this yet?

**Comment by Lauren34** [comment]
So sorry, just did

**Pull request merged**